hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/FilteringConfiguration.java

```diff
@@ -20,7 +21,8 @@
     FilterChainFactory filterChainFactory() {
         List<SubscriptionMessageFilterCompiler> subscriptionFilterCompilers = Arrays.asList(
                 new AvroPathSubscriptionMessageFilterCompiler(),
-                new JsonPathSubscriptionMessageFilterCompiler()
+                new JsonPathSubscriptionMessageFilterCompiler(),
+                new HeaderSubscriptionMessageFilterCompiler()
```

Comment:
As far as I remember, hermes-management is aware of the filters only because it has a logic related to the filters debugger. As the debugger doesn't currently support header filters, I think this change is not needed.

Reply:
You are right, this change is not needed. However I added it to simplify adding support for headers dubbuger in the future. Thanks to this line only changes on frontend side are needed now and next developer don't have to analyze backend side at all.